components/brave_referrals/browser/BUILD.gn

```diff
@@ -3,9 +3,7 @@ import("//brave/components/brave_referrals/buildflags/buildflags.gni")
 source_set("browser") {
   # Remove when https://github.com/brave/brave-browser/issues/10646 is resolved
   check_includes = false
-  public_deps = [
-    "//brave/components/brave_referrals/buildflags",
-  ]
+  public_deps = [ "//brave/components/brave_referrals/buildflags" ]
```

Comment:
please avoid unrelated changes, it makes it more difficult to review the PR

Reply:
I believe this is because of running `git cl format` over the whole repo.

Reply:
I would still to just the files you are working on and leave the gn files alone. It makes it hard to see what's actually changing and also it's going to make the PR a rebase nightmare for you

Reply:
I can revert those changes then

Reply:
- [ ] revert formatting changes in unrelated files